Makefile

```diff
@@ -32,6 +33,16 @@ all: Makefile .docker-image conf
 	cp ./update_default_resources.c ${POKY}/meta-pelion-edge/recipes-wigwag/mbed-edge-core/files/
 	$(call docker_run, make bb/${IMAGE_RECIPE})
 
+flash-%: all
+	$(foreach dev, $(wildcard /dev/${*}*),\
+		sudo umount ${dev} || true; \
+	)
+	sudo bmaptool copy \
```

Comment:
Is it possible to make this fall back to 'dd' if the user doesn't have bmaptool installed?

Reply:
Shouldn't the tool be guaranteed to exist since this being run within a docker context?

Reply:
This recipe doesn't appear to be executed from a Docker context.  Check the "bitbake-%" and related "bb/%" recipes below for an example of recipes in this Makefile that run within Docker.

Reply:
Ahh, I see. My mistake.

Reply:
Sure, we can have this fallback. I'll update this PR shortly.